core/sr-io/without_std.rs

```diff
@@ -678,10 +780,18 @@ impl StorageApi for () {
 		}
 	}
 
-	fn exists_child_storage(storage_key: &[u8], key: &[u8]) -> bool {
+	fn exists_child_storage(child_trie: ChildTrieReadRef, key: &[u8]) -> bool {
+		let empty_byte: [u8;0] = [];
+		let (keyspace, root) = match child_trie {
+			ChildTrieReadRef::New(keyspace) => (keyspace, &empty_byte[..]),
```

Comment:
nit: I'm pretty sure `(keyspace, &[])` works for creating an empty byte slice inline.

Reply:
yes static